spec/integration/alerts_spec.rb

```diff
@@ -286,6 +286,13 @@ module Hawkular::Alerts::RSpec
       expect(events).to_not be_nil
       expect(events.size).to be(0)
     end
+
+    it 'Should create an event' do
```

Comment:
Minor comment.
Perhaps a "describe" section for events ?
I'm also touching this file and to not mixed cassetes, splitting on "describe" could be nice.